app/src/androidTest/java/com/sdp/movemeet/map/MiniMapFragmentTest.java

```diff
@@ -47,11 +47,13 @@
     private FirebaseAuth fAuth;
     private FirebaseUser user;
 
+
```

Comment:
Why this class is full commented ?

Reply:
It didn't work for some reason. But I wanted to first merge everything I did before trying to solve this problem. I still intend to make it work at some point haha